src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -112,30 +114,34 @@
 					Style="{StaticResource PropertyValueTextBlock}"
 					Text="{x:Bind ViewModel.ItemType, Mode=OneWay}" />
 
+				<!--  File System  -->
 				<TextBlock
 					x:Name="FileSystem"
 					Grid.Row="2"
 					Grid.Column="0"
-					Padding="4,0,0,0"
+					Margin="4,0,0,0"
 					x:Load="{x:Bind ViewModel.DriveFileSystemVisibility, Mode=OneWay}"
 					FontWeight="SemiBold"
 					Style="{StaticResource PropertyName}"
 					Text="{helpers:ResourceString Name=PropertiesDriveFileSystem/Text}" />
+
 				<TextBlock
 					x:Name="DriveFileSystemValue"
 					Grid.Row="2"
 					Grid.Column="1"
 					x:Load="{x:Bind ViewModel.DriveFileSystemVisibility, Mode=OneWay}"
 					Style="{StaticResource PropertyValueTextBlock}"
 					Text="{x:Bind ViewModel.DriveFileSystem, Mode=OneWay}" />
+
 			</Grid>
 
-			<!--  Item details  -->
+			<!--  Item Details  -->
 			<Expander
 				x:Name="ItemDetails"
 				HorizontalAlignment="Stretch"
 				HorizontalContentAlignment="Stretch"
 				x:Load="{x:Bind ViewModel.ItemPathVisibility, Mode=OneWay}"
+				CornerRadius="8"
```

Comment:
Is this is for consistency with the change on line 48?

Reply:
Yes

Reply:
Looks good, thanks!